src/Context.h

```diff
@@ -55,11 +55,12 @@ namespace sua {
         std::shared_ptr<IMqttMessagingProtocol> messagingProtocol;
         std::shared_ptr<IMqttProcessor>         mqttProcessor;
         std::shared_ptr<IBundleChecker>         bundleChecker;
-        std::string                             updatesDirectory    = "/data/selfupdates";
-        std::string                             tempFileName        = "/temp_file";
-        std::string                             certificateFileName = "/etc/ssl/certs/ca-certificates.crt";
-        bool                                    downloadMode        = true;
-        bool                                    fallbackMode        = false;
+        std::string                             updatesDirectory = "/data/selfupdates";
+        std::string                             tempFileName     = "/temp_file";
+        std::string                             caPath           = "/etc/ssl/certs";
+        std::string                             caFile           = "/etc/ssl/certs/ca-certificates.crt";
```

Comment:
Init. of updatesDirectory, caPath and caFile could be empty, as these are is set in main().

Reply:
Fixed